### DISCUSSION TITLE
Privacy Policy

### DISCUSSION THREAD

**Discussion by FibreTTP** [CappielloAntonio/tempo#284] [Q&A] [answered]
The [Privacy Policy](https://github.com/CappielloAntonio/tempo/blob/e465892013ddd5d08ba1eae0989bd631b77b0c0e/privacy.html) added in e465892 explicitly states that information about the device is collected, including and presumably not limited to:

- "Your device's Internet Protocol address (e.g. IP address)"
- "The pages of the Application that you visit, the time and date of your visit, the time spent on those pages"
- "The time spent on the Application"
- "The operating system you use on your mobile device"

It is also stated that the information may be used to provide us with "marketing promotions", and that the only way to opt-out is to uninstall the application.

The policy states that it was generated with [App Privacy Policy Generator](https://app-privacy-policy-generator.nisrulz.com), but with the way that generator works, the creator can choose a "no tracking" option which does not output the text above.

This policy is not presented to the user on first launch, nor is it available on the [website](https://cappielloantonio.github.io/tempo).

Can you please confirm that Tempo *does* collect this information, and for what purpose it is used for?

**Comment by CappielloAntonio** [comment, marked as answer]
The privacy policy certainly needs to be reviewed, it was automatically generated using that generator that you linked and is intended exclusively for publication on the Play Store.

This "rush" was caused by Google's threat of termination of my developer account due to inactivity. I sent the app for review and published it with limited access on the Play Store so as not to be kicked out and therefore lose the registration money.

The information collected is not collected directly by the app, but during the download from the Play Store.

Forgive me for the confusion, it will be fixed as soon as possible.

**Reply by FibreTTP** [reply]
Thanks. I haven't heard good things about publishing apps on the Play Store.

Please update and mark as answer when it is remade :)

**Reply by CappielloAntonio** [reply]
I just committed a clarification to the privacy policy html page, and it will be released shortly.

Thank you for pointing out the problem, and I apologize again for the confusion.

More changes to the policy content will likely follow.
